helm/openwhisk/runtimes.json

```diff
@@ -34,6 +34,20 @@
                     "attachmentName": "codefile",
                     "attachmentType": "text/plain"
                 }
+            },
+            {
+                "kind": "nodejs:10",
+                "default": false,
+                "image": {
+                    "prefix": "openwhisk",
+                    "name": "action-nodejs-v10",
+                    "tag": "d967ebe"
```

Comment:
This tag doesn't exist in dockerhub for action-nodejs-v10.  Current available tags are `e892cd8` or `586f84e`.

Reply:
it was a copy paste error from nodejs8, Are we pinning now?
You don't like to leave on the bleeding edge?

Reply:
Thanks.  Yeah, the chart is much more useful if the deployments using it are repeatable.  So I've  switched to pinning all docker tags & git clones, with a plan of updating with known good versions on a frequent basis (roughly weekly...).